tests/bnb_kbit/test_mixed_int8.py

```diff
@@ -77,11 +77,11 @@ class BaseMixedInt8Test(unittest.TestCase):
 
     # Constant values
     EXPECTED_RELATIVE_DIFFERENCE = (
-        1.540025  # This was obtained on a Quadro RTX 8000 so the number might slightly change
+        1.054829776346287  # This was obtained on a Quadro RTX 8000 so the number might slightly change
```

Comment:
Hmm I think this shouldn't be changed, you probably tested on another device than a T4?

Reply:
Ok after some investigaation it seems to be related to https://github.com/huggingface/transformers/pull/23479/files#r1200368398
Could you please revert this change, and just change the way `self.model_8bit` is loaded in `setUp` to load it with `torch_dtype=torch.float16` ?